### PR TITLE
Update search.js

### LINKED ISSUE #4179: [OTHER] Change the alert for the search page
### What would you like to share?

search

Changing "search for users" to "search for users, tags or languages."

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by khalidx3**
## Fixes Issue

Closes #4179 

## Changes proposed
Search user changed to Search users,  tags or languages.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4197"></a>

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ export default function Search({ users }) {
         {!threeOrMore && (
           <Alert
             type="info"
-            message="You have to enter at least 3 characters to search for a user."
+            message="You have to enter at least 3 characters to search for a users,tags or languages."
```
Comment (swagataroy30) [inline pages/search.js:115]:
```suggestion
            message="You have to enter at least 3 characters to search for users,tags or languages."
```

**Review by swagataroy30 — COMMENTED** [review]
'a' may not be required with plurals.

File: pages/search.js (hunk 1 above)
Comment (khalidx3) [reply pages/search.js]: Thanks @swagataroy30 for pointing out the mistake it has been updated

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**